### PR TITLE
Seed the agenda with the current contents of #100

### PR DESCRIPTION
**Pull request by chowdhurian**
This PR simply contains the current contents of the original comment in #100.

Reason: https://github.com/nodejs/summit/issues/100#issuecomment-422465874

### CODE REVIEW

File: 2018-10-Vancouver/agenda.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+| 3:30          | Main         | Moderation                                    | @hackygolucky                          | |
+| 3:30          | Main         | Mentorship                                    | @Bamieh                                | |
+| 3:30          | Main         | LTS + release                                          |    @MylesBorins          | https://github.com/nodejs/Release/issues/358  |
+| 3:30          | Main         | Free                                          |                                        | |
+| 3:30          | Main         | Free                                          |                                        | |
+| 3:30          | Breakout 1   | Diagnostics                                   | @yunong                                | |
+| 3:30          | Breakout 2   | Promises                                      | @BridgeAR                              | |
+
+
+_If you want to speak and present a topic in the main room, feel free to reply to this issue with a title and abstract. If you want to propose a full session in a breakout room, add a link to an issue in this repo, similar to https://github.com/nodejs/summit/issues/86_.
```
Comment (joyeecheung) [inline 2018-10-Vancouver/agenda.md]: Can you change this from “reply to this issue” to “send a PR to update this document”?

File: 2018-10-Vancouver/agenda.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+<br>
+
+
+| Time          | Place        | Topic                                                               | Speakers/Champions            | Issue                                      |
+|---------------|--------------|---------------------------------------------------------------------|-------------------------------|--------------------------------------------|
+| 8:00          |              | Breakfast and registration                                          |                               |                                            |
+| 9:00          | Main         | Intro, Code of Conduct                                              | @mcollina @dshaw @chowdhurian |                                            |
+| 9:15          | Main         | Modules                                                             | @MylesBorins                  | https://github.com/nodejs/modules/issues/177 | 
+| 9:15          | Breakout 1   | Website Redesign                                                    | @AdamMiller                   | https://github.com/nodejs/website-redesign/issues/88 |
```
Comment (thefourtheye) [inline 2018-10-Vancouver/agenda.md]: Isn't this supposed to be amiller-gh?

**Comment by mhdawson** [comment]
I made a few updates to the original copy.  Would it be possible to integrate them?

Might also make sense to make a bold comment above the original saying the agenda's being maintained here instead?

**Comment by chowdhurian** [comment]
Thanks for the comments, folks! 🙏🏽

 - [x] bring in the new updates from #100.
 - [x] resolve comments
 - [] make a bold comment above the original saying the agenda's being maintained here instead

**Comment by chowdhurian** [comment]
I'm going to merge this, so it's ready for edits asap.

**Pull request merged**